BeardedManStudios/Source/Forge/Networking/NetWorker.cs

```diff
@@ -1040,7 +1040,7 @@ public static void RefreshLocalUdpListings(ushort portNumber = DEFAULT_PORT, int
 
 				try
 				{
-					while (localListingsClient != null && !EndingSession)
```

Comment:
Rather than removing the !EndingSession, can you make this `do { ... } while (localListingsClient != null && !EndingSession)`.

The reason for this request is because though this will resolve the immediate problem, this may cause a thread to be lost and cause force close to be required in some cases (people will think it is a crash when trying to close the application). This might also cause some weirdness in Unity between play modes.